core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineDb2.java

```diff
@@ -262,6 +262,18 @@ public String tableExistsInternalTableCheckSql(String fullTableName) {
     return "SELECT 1 FROM " + fullTableName + " LIMIT 1";
   }
 
+  @Override
+  public String createIndexSql(
+      String schema, String table, String indexName, String indexedColumn) {
+    return "CREATE INDEX "
```

Comment:
How about moving this default implementation that also exists in other RdbEngines except for Oracle to RdbEngineStrategy as a `default` implementation?